packages/cli/package.json

```diff
@@ -2,23 +2,14 @@
   "name": "@chainflip-io/cli",
   "version": "0.0.9",
   "description": "",
-  "main": "./dist/lib.js",
-  "types": "./dist/lib.d.ts",
   "bin": {
     "chainflip": "./dist/cli.js"
   },
   "exports": {
     ".": {
       "import": "./dist/lib.mjs",
       "require": "./dist/lib.js",
-      "type": "./dist/lib.d.ts"
-    }
-  },
-  "typesVersions": {
-    "*": {
-      ".": [
-        "dist/lib.d.ts"
-      ]
+      "types": "./dist/lib.d.ts"
```

Comment:
`exports.types` is preferred over `typesVersions` since typescript 4.9: https://stackoverflow.com/a/75738953